src/Castle.Windsor/Core/LifestyleType.cs

```diff
@@ -45,12 +45,11 @@ public enum LifestyleType
 		///   instance in a pool instead of always creating them.
 		/// </summary>
 		Pooled,
-#if !(SILVERLIGHT)
```

Comment:
We should restore this conditional compilation but against `FEATURE_SYSTEM_WEB` since the module has that. Only saw 2 other files use it which used to have !SILVERLIGHT:

- LifestyleModelInspector.cs
- PotentialLifestyleMismatchesDiagnostic.cs